### DISCUSSION TITLE
iOS安装脚本 可否加一个网页按钮进入设置页面

### DISCUSSION THREAD

**Discussion by wodahsnk** [F9y4ng/GreasyFork-Scripts#221] [问答专区(Question / Answer)]
iOS安装脚本 可否加一个网页按钮进入设置页面

**Comment by F9y4ng** [comment]
iOS手机？脚本并没有对手机移动端进行兼容。如果是MacOS的safari，可以使用快捷键。

**Comment by wodahsnk** [comment]
![IMG_2186](https://github.com/F9y4ng/GreasyFork-Scripts/assets/24026493/5f73e0d6-bbbb-48a3-82a1-7730c5ada874)
iOS手机也可以安装 希望可以加个按钮进入设置页面  电脑把按钮隐藏就行 不知道开发者可以兼容一波吗

**Comment by F9y4ng** [comment]
几乎所有的浏览器，在拥有脚本管理器的情况下都可以安装第三方用户脚本。但，脚本是否兼容操作系统，是否兼容浏览器，是否兼容脚本管理器，是否兼容你要访问的网站才是重点。

此外，从目前来看，几乎所有的移动端设备分辨率都是高分辨率、高刷新率，也不存在字体渲染的问题，所以完全没必要多次一举。

所以，你的这个需求，应该不会被采纳。如果你仅需要更换字体，建议你可以寻找其他的扩展或脚本尝试。
